src/main/resources/static/js/popup/ConnectBase.js

```diff
@@ -0,0 +1,23 @@
+class ConnectBase {
+  constructor() {
+    this._url = "http://localhost:3000/auth/";
+    this.headers = {
+      "Content-Type": "application/json"
+    };
+  }
+
+  async postRequest(body, value) {
+    let url;
```

Comment:
You may initialize this variable by `this._url` and add "login" or "register" in line 12 and 14 as well.